jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.py

```diff
@@ -101,7 +103,8 @@ def find_public_metadata(self, meta, primary_only=False):
         d = flatten_nested_dict(meta)
         # Some FITS keywords cause "# ipykernel cannot clean for JSON" messages.
         # Also, we want to hide internal metadata that starts with underscore.
-        badkeys = ['COMMENT', 'HISTORY', ''] + [k for k in d if k.startswith('_')]
+        badkeys = (['COMMENT', 'HISTORY', ''] + [k for k in d if k.startswith('_')]
+                   + [k for k in d if k.startswith('original_')])

```

Comment:
Without this, I get things in Cubeviz that is not from original header like:

* original_spectral_axis_unit (a Quantity)
* original_wcs (WCS object)

Perhaps these should have been hidden with leading underscore but they snuck through so here we are.

Reply:
Weird thing is I don't know where these are being injected. A simple grep in the code base gave me no match.

Reply:
Oh yeah, I think @kecnry or @rosteen was right. These are from specutils

https://github.com/astropy/specutils/blob/170d70f4d3d0e9db624b1346ae97b5ad5f7b2fbe/specutils/spectra/spectrum_mixin.py#L243-L248